pyregence/gridfire
# GRID1-392 Update elm_to_grid.clj to emit the grid of rasters input type for each input layer

-------

## Purpose

## Related Issues
Closes GRID-338

## Submission Checklist
- [ ] Commits include the JIRA issue and the `#review` hashtag (e.g. `GRID-### #review <comment>`)

## Testing

1. Go to resources folder and run:

```
./elm_to_grid.clj --elmfire-config ../test/gridfire/resources/tiled_io_elmfire.data
```

2. Check the grid of input layers is properly setup
